tsdb/engine_test.go

```diff
@@ -559,29 +559,29 @@ func TestProcessRawQueryDerivative(t *testing.T) {
 		name     string
 		fn       string
 		interval time.Duration
-		in       []*rawMapperValue
-		exp      []*rawMapperValue
+		in       []*mapperValue
+		exp      []*mapperValue
 	}{
 		{
 			name:     "empty input",
 			fn:       "derivative",
 			interval: 24 * time.Hour,
-			in:       []*rawMapperValue{},
-			exp:      []*rawMapperValue{},
+			in:       []*mapperValue{},
+			exp:      []*mapperValue{},
 		},
 
 		{
 			name:     "single row returns 0.0",
 			fn:       "derivative",
 			interval: 24 * time.Hour,
-			in: []*rawMapperValue{
-				&rawMapperValue{
+			in: []*mapperValue{
+				&mapperValue{
 					Time:  time.Unix(0, 0).Unix(),
 					Value: 1.0,
 				},
 			},
```

Comment:
You can remove the `&mapperValue` since it's implicit from the slice type:

```go
in: []*mapperValue{
    {
        Time: time.Unix(0, 0).Unix(),
        Value: 1.0,
    },
}
```